webui/react/src/utils/observable.ts

```diff
@@ -26,7 +25,7 @@ const useValueMemoizedObservable = <T>(o: Observable<T>): T => {
       forceRender({});
     }
     return o.subscribe((value, prevValue) => {
-      if (!isEqual(value, prevValue)) {
+      if (!_.isEqual(value, prevValue)) {
```

Comment:
I was unaware at the time that `isEqual` is incorrect. This function relies on a working `isEqual` method to work correctly so this is a bugfix. Noticed via another PR that we already have `lodash` in the build.